portfolio/src/main/webapp/index.html

```diff
@@ -2,14 +2,27 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Dhruvi's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
+      <h1>Dhruvi's Portfolio</h1>
+      <p>This is Dhruvi's portfolio.</p>
+      <p>Hi there! My name is Dhruvi and I am a STEP intern @ Google! I study CS at Rutgers University-New Brunswick!</p>
+      <div class="icons"> 
+            <a href="https://www.linkedin.com/in/dhruvi-sheth/">
+                <img alt="Linkedin" src="images/linkedin.png" width="30" height="25">
```

Comment:
nit: Any particular reason not to make it a square?

Reply:
only for aesthetic reason since the logo itself is not a square

Reply:
Thanks for the clarification